projects/addon-doc/package.json

```diff
@@ -7,25 +7,26 @@
     "license": "Apache-2.0",
     "repository": "https://github.com/tinkoff/taiga-ui",
     "homepage": "https://github.com/tinkoff/taiga-ui",
+    "dependencies": {
+        "markdown-it": "^12.3.2",
+        "highlight.js": "^11.3.1",
+        "ngx-highlightjs": "^4.1.4",
+        "schematics-utilities": "^1.0.1",
+        "@angular-devkit/schematics": "^9.0.0"
+    },
+    "devDependencies": {
+        "@types/markdown-it": "^12.2.3"
+    },
     "peerDependencies": {
         "@angular/common": ">=9.0.0",
         "@angular/core": ">=9.0.0",
         "@angular/forms": ">=9.0.0",
         "@angular/router": ">=9.0.0",
-        "@angular-devkit/schematics": "^9.0.0",
-        "@ng-web-apis/common": "1.12.1",
-        "@tinkoff/ng-polymorpheus": "3.1.12",
-        "@taiga-ui/cdk": "2.31.0",
-        "@taiga-ui/core": "2.31.0",
-        "@taiga-ui/kit": "2.31.0",
-        "@taiga-ui/addon-mobile": "2.31.0",
-        "schematics-utilities": "1.0.1",
-        "highlight.js": "^10.0.0",
-        "ngx-highlightjs": "^4.0.0"
-    },
-    "devDependencies": {
-        "@taiga-ui/core": "2.31.0",
-        "@taiga-ui/kit": "2.31.0",
-        "@taiga-ui/addon-mobile": "2.31.0"
+        "@ng-web-apis/common": ">=1.12.1",
+        "@tinkoff/ng-polymorpheus": ">=3.1.12",
```

Comment:
@splincode `addon-doc` doesn't seem to be compatible with `ng-polymorpheus` v4. So, it might be best to keep this one at `^3.1.12`.

Reply:
@hakimio Hello, It's just peerDependencies, but if you have `@taiga-ui/cdk`, we will install the correct version from the dependencies

![image](https://user-images.githubusercontent.com/12021443/151938330-863504f7-6df1-48f9-8711-0017eb9c48bd.png)

Reply:
@splincode how is `@taiga-ui/cdk` related to `@tinkoff/ng-polymorpheus`? I don't follow you...

Reply:
before using the `addon-doc`, you will install third party packages anyway

```
$ npm i @taiga-ui/{cdk,core,kit, addon-mobile}
```

Nx graph dependencies

![image](https://user-images.githubusercontent.com/12021443/151939822-56e043d3-ce4e-4449-b250-42b03f67929e.png)

And in the cdk package we have a dependency of only the 3rd major version

Reply:
Yes, `cdk` has a "hard dependency" on `ng-polymorpheus` `v3`, but that doesn't change the fact that if you install `ng-polymorpheus` `v4` in your own project using `addon-doc`, it breaks. Try it out if you don't believe me.

You can try with the following deps:
```json
"dependencies": {
    "@angular/animations": "^13.2.0",
    "@angular/cdk": "^13.2.0",
    "@angular/common": "^13.2.0",
    "@angular/core": "^13.2.0",
    "@angular/forms": "^13.2.0",
    "@angular/localize": "^13.2.0",
    "@angular/material": "^13.2.0",
    "@angular/platform-browser": "^13.2.0",
    "@angular/platform-browser-dynamic": "^13.2.0",
    "@angular/router": "^13.2.0",
    "@ng-web-apis/common": "^2.0.0",
    "@ng-web-apis/mutation-observer": "^1.1.0",
    "@taiga-ui/addon-doc": "^2.32.0",
    "@taiga-ui/addon-mobile": "^2.32.0",
    "@taiga-ui/cdk": "^2.32.0",
    "@taiga-ui/core": "^2.32.0",
    "@taiga-ui/i18n": "^2.32.0",
    "@taiga-ui/icons": "^2.32.0",
    "@taiga-ui/kit": "^2.32.0",
    "@tinkoff/ng-event-plugins": "^2.5.1",
    "@tinkoff/ng-polymorpheus": "^4.0.0",
    "highlight.js": "^11.4.0",
    "ngx-bootstrap": "^8.0.0",
    "ngx-highlightjs": "^4.1.4",
    "rxjs": "^7.5.2",
    "tslib": "^2.3.1",
    "zone.js": "~0.11.4"
  }
```

Reply:
BTW, I am using `yarn` instead of `npm`.

Reply:
`"@tinkoff/ng-polymorpheus": "^4.0.0",` If you manually specify this dependency in your project package.json, our `peerDependencies` will not help you, just remove it from the dependencies (project package.json) and don't worry!

Reply:
`npm` installs peer dependencies automatically, but if you use `yarn`, you have to do it manually. Anyway, if you don't want to do anything about this, it's fine. It's not really important to me.

Reply:
@hakimio please, provide minimal reproduction github repo with `@taiga-ui/2.32.0` where it won't work.

Reply:
[https://github.com/hakimio/ngx-ui-tour](Github repo). Use `yarn` (v1) package manager and [update `ng-polymorpheus`](https://github.com/hakimio/ngx-ui-tour/blob/eebbee87919798722083d36fa28efd19e3999daf/package.json#L70) to `v4`.